partner_credit_limit_history/wizard/partner_credit_limit_wizard.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class PartnerCreditLimitWizard(models.TransientModel):
+    _name = "partner.credit.limit.wizard"
+    _description = "Partner credit limit wizard"
+
+    partner_id = fields.Many2one(
+        comodel_name="res.partner", string="Partner", required=True
+    )
+    amount = fields.Float(string="New Amount", required=True)
+    note = fields.Char(string="Comments", required=True)
```

Comment:
Why is the comment necessary?

Reply:
Maybe the string can be `Reason` for better understanding of the requirement